great_expectations/self_check/util.py

```diff
@@ -1193,58 +1226,58 @@ def candidate_test_is_on_temporary_notimplemented_list_cfe(context, expectation_
-            # "expect_column_sum_to_be_between",
             "expect_column_pair_values_A_to_be_greater_than_B",
             "expect_column_pair_values_to_be_equal",
             "expect_column_pair_values_to_be_in_set",
             "expect_multicolumn_values_to_be_unique",
             "expect_multicolumn_sum_to_equal",
             "expect_column_pair_cramers_phi_value_to_be_less_than",
-            # "expect_column_kl_divergence_to_be_less_than",
+            "expect_column_bootstrapped_ks_test_p_value_to_be_greater_than",
+            "expect_column_chisquare_test_p_value_to_be_greater_than",
+            "expect_column_parameterized_distribution_ks_test_p_value_to_be_greater_than",
+            "expect_compound_columns_to_be_unique",
+        ]
+
+    if context == "bigquery":
+        ###
+        # NOTE: 20210729 - jdimatteo: It is relatively slow to create tables for
+        # all these tests in BigQuery, and if you want to run a single test then
+        # you can uncomment and modify the below line (which results in only the
+        # tests for "expect_column_values_to_not_be_null" being run):
+        # return expectation_type != "expect_column_values_to_not_be_null"
+        ###
+        # NOTE: 20210729 - jdimatteo: Below are temporarily not being tested
+        # with BigQuery. For each disabled test below, please include a link to
+        # a github issue tracking adding the test with BigQuery.
+        ###
+        return expectation_type in [
+            "expect_column_kl_divergence_to_be_less_than",  # TODO: Takes over 64 minutes to "collect" (haven't actually seen it complete yet) -- follow up / triage with https://github.com/great-expectations/great_expectations/issues/3132.
+            "expect_column_values_to_be_in_set",  # TODO: No matching signature for operator and AssertionError: expected ['2018-01-01T00:00:00'] but got ['2018-01-01'] -- follow up / triage with https://github.com/great-expectations/great_expectations/issues/3132.
+            "expect_column_values_to_be_in_type_list",  # TODO: AssertionError, follow up / triage with https://github.com/great-expectations/great_expectations/issues/3132.
+            "expect_column_values_to_be_between",  # TODO: "400 No matching signature for operator >=" -- follow up / triage with https://github.com/great-expectations/great_expectations/issues/3132.
+            "expect_column_quantile_values_to_be_between",  # TODO: takes over 15 minutes to "collect" (haven't actually seen it complete yet) -- follow up / triage with https://github.com/great-expectations/great_expectations/issues/3132.
+            "expect_column_mean_to_be_between",  # TODO: "400 No matching signature for operator *" -- follow up / triage with https://github.com/great-expectations/great_expectations/issues/3132.
+            "expect_select_column_values_to_be_unique_within_record",  # Starting here, the list is common to list expectations that are not working for all db backends
```

Comment:
These comments are amazing. Thanks again @jdimatteo 🎉 

Should we have these in Confluence as well? I forgot what the path forward was for these expectations.

Reply:
Can we please create one or more issues to track fixing these 7 tests that work on other db backends, and update these comments to point to them?

https://github.com/great-expectations/great_expectations/issues/3132 will be closed by this PR, and I think it would be good to create separate issues to help ensure the others are followed up on.

Reply:
Created two issues #3260 and #3261. The expectations are being tracked as individuals JIRA tickets internally